.github/FUNDING.yml

```diff
@@ -0,0 +1 @@
+open_collective: ramda
```

Comment:
I _think_ this will show up once we enable this:

![](https://docs.github.com/assets/cb-36772/images/help/sponsors/sponsorships-checkbox.png)

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/displaying-a-sponsor-button-in-your-repository